### DISCUSSION TITLE
Just starting, help with Region/Site/Tenant initial build

### DISCUSSION THREAD

**Discussion by Talmars04** [netbox-community/netbox#18161] [Help Wanted!]
Hello all,

Just starting off with NetBox.  I've watched a handful of youtube's about Regions, sites, tenants and all seem to come from an angle of some huge multinational company or MSP etc.  

Does anyone have a good example of building out a campus like setup with multiple buildings/floors.   Take where I'm at for example.  I'm at a Healthcare Organization, 2 separate Hospital Campuses, each campus with many different departments on multiple floors.  Each campus also has exterior buildings that directly connect to the main hospital with other departments.  I also have many clinics all around the area in neighboring towns that connect back to the main hospital datacenter via VPN tunnels.

You could also use a University as an example.  1 organization, 1 geographical location but many buildings and many floors in those buildings with multiple departments in those buildings/floors.

Any help would be greatly appreciated.  I think once I get this part down I'll be good to go.

Thanks all.

**Comment by a084ed22** [comment]
There is really no good or bad way to go about it as it really depends on the level of detail you want to capture. In a somewhat similar setup we chose to model regions and sites, but we do not delve in detail such as building floors or rooms, nor we track departments and their mapping.
